browser/BUILD.gn

```diff
@@ -245,6 +245,9 @@ source_set("browser_process") {
       "binance/binance_service_factory.h",
       "binance/binance_protocol_handler.cc",
       "binance/binance_protocol_handler.h",
+      "binance/binance_util.cc",
+      "binance/binance_util.h",
+      "static_values.h",
```

Comment:
I think this should be `binance/static_values.h`?

Reply:
thanks for catching, fixed